css-typed-om/Overview.bs

```diff
@@ -3509,58 +3509,140 @@ regardless of what property it is for.
+        and return the result.
 
 : <a>font-max-size</a>
-::
+:: For both specified and computed values:
+
+    1. If the value is an <<absolute-size>>, <<relative-size>> or <css>infinity</css>, 
+        [=reify an identifier=] from the value
+        and return the result.
+
+    2. Otherwise, [=reify a numeric value=] from the value
+        and return the result.
 
 : <a>font-min-size</a>
-::
+:: Same as 'font-size'
 
 : <a>font-optical-sizing</a>
-::
+:: For both specified and computed values,
+    [=reify an identifier=] from the value
+    and return the result.
 
 : <a>font-palette</a>
-::
+:: For both specified and computed values:
+
+    1. If the value is <css>normal</css>, <css>light</css> or <css>dark</css>,
+        [=reify an identifier=] from the value
+        and return the result.
+
+    2. Otherwise, reify as a {{CSSStyleValue}}
+        and return the result.
 
 : <a>font-presentation</a>
-::
+:: For both specified and computed values,
+    [=reify an identifier=] from the value
+    and return the result.
 
 : <a>font-size</a>
-::
+:: For both specified and computed values:
+
+    1. If the value is an <<absolute-size>> or <<relative-size>>, 
+        [=reify an identifier=] from the value
+        and return the result.
+
+    2. Otherwise, [=reify a numeric value=] from the value
+        and return the result.
 
 : <a>font-size-adjust</a>
-::
+:: For both specified and computed values:
+
+    1. If the value is <css>none</css>, 
+        [=reify an identifier=] from the value
+        and return the result.
+
+    2. Otherwise, [=reify a numeric value=] from the value
+        and return the result.
 
 : <a>font-stretch</a>
-::
+:: For both specified and computed values:
+
+    1. If the value is a <<percentage>>, 
+        [=reify a numeric value=] from the value
+        and return the result.
+
+    2. Otherwise, [=reify an identifier=] from the value
+        and return the result.
 
 : <a>font-style</a>
-::
+:: For both specified and computed values:
+
+    1. If the value is <css>normal</css> or <css>italic</css>, 
+        [=reify an identifier=] from the value
+        and return the result.
+
+    2. Otherwise, reify as a {{CSSStyleValue}}
+        and return the result.
 
 : <a>font-synthesis</a>
-::
+:: For both specified and computed values:
+
+    1. If the value is <css>none</css>, <css>weight</css>, <css>style</css> or <css>small-caps</css>,
+        [=reify an identifier=] from the value
+        and return the result.
+
+    2. Otherwise, reify as a {{CSSStyleValue}}
+        and return the result.
 
 : <a>font-variant</a>
-::
+:: For both specified and computed values,
+    reify as a {{CSSStyleValue}}
+    and return the result.
 
 : <a>font-variant-alternates</a>
 ::
 
 : <a>font-variant-emoji</a>
-::
+:: For both specified and computed values,
+    [=reify an identifier=] from the value
+    and return the result.
 
 : <a>font-variation-settings</a>
-::
+:: For both specified and computed values:
```

Comment:
The syntax is `none | <something>#`, so it's neither list-valued nor single-valued. I think reification-wise it's fine, but the wording for `styleMap.append` etc. might be hard (can append `<something>` if it's not `none`, can never append `none`). Alternatively, we can always represent the list values as a separate type (like `transform` which suffers the same problem), but it means we can never `append` to it directly.

Reply:
Ah, I figured we probably had an example of this somewhere; the 'animation' handling of "none" seemed sufficiently weird that I was pretty sure *somewhere* wouldn't have gone for it.

Reply:
(It's "normal", not "none".)

Let's leave this off for the moment - we can just reify to a CSSStyleValue until we figure it out. I suspect this'll be something we want to run by the group and get opinions on.